client/post-editor/editor-sharing/publicize-message.jsx

```diff
@@ -88,13 +107,15 @@ class PublicizeMessage extends Component {
 			);
 		} else {
```

Comment:
Nitpick and unrelated to your changes but this would be a good chance to get rid of this `else` that eslint complains about and just:
```
if () {
  return;
}
return;
```

Also fine if you don't wanna bother with that. :-)